mlir/test/Integration/lit.local.cfg

```diff
@@ -58,3 +51,61 @@ aarch64_mcr_cmd = configure_aarch64_mcr_cmd()
 # * ARM_EMULATOR_UTILS_LIB_DIR - AArch64 native utilites library to support
 #                                cross-compilation.
```

Comment:
The comment on L44-L52 can be moved above this line:
```python
aarch64_mcr_cmd = configure_aarch64_mcr_cmd()
```

Basically, everything that follows is in preparation for possible emulation. I would also move the block that starts with `if config.arm_emulator_utils_lib_dir:` into a function to match `configure_aarch64_mcr_cmd`.

This will most likely require some updates to the comments, but will lead to a much cleaner config.

Reply:
Thanks for the suggestion, I've refactored the substitutions.